electron/ipc/register/project.ts

```
@@ -0,0 +1,376 @@
+      if (!filePath) {
+        return { success: false, error: 'Only auto-generated recordings can be deleted' };
+      }
+      const resolvedPath = await fs.realpath(filePath).catch(() => path.resolve(filePath));
+      const recordingsDir = await getRecordingsDir();
+      if (!isPathInsideDirectory(resolvedPath, recordingsDir) || !isAutoRecordingPath(resolvedPath)) {
+        return { success: false, error: 'Only auto-generated recordings can be deleted' };
+      }
+      await fs.unlink(resolvedPath);
+      // Also delete the cursor telemetry sidecar if it exists
+      const telemetryPath = getTelemetryPathForVideo(resolvedPath);
+      await fs.unlink(telemetryPath).catch(() => {});
```

Biome (2.4.11)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)